src/Umbraco.Cms.ManagementApi/Controllers/MemberType/Tree/MemberTypeTreeControllerBase.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Umbraco.Cms.Core;
+using Umbraco.Cms.Core.Models;
+using Umbraco.Cms.Core.Models.Entities;
+using Umbraco.Cms.Core.Services;
+using Umbraco.Cms.ManagementApi.ViewModels.Tree;
+using Umbraco.New.Cms.Web.Common.Routing;
+
+namespace Umbraco.Cms.ManagementApi.Controllers.MemberType.Tree;
+
+[ApiVersion("1.0")]
+[ApiController]
+[VersionedApiBackOfficeRoute($"{Constants.UdiEntityType.MemberType}/tree")]
+public class MemberTypeTreeControllerBase : TreeControllerBase<TreeItemViewModel>
+{
+    private readonly IContentTypeService _contentTypeService;
```

Comment:
I'm not entirely sure why this takes and saves  `IContentTypeService` and then proceeds to never use it. Maybe just an oversight? Or am I missing something 😄

Reply:
Was an oversight, corrected in vLatest 👍